protos/shared/shared.proto

```diff
@@ -29,9 +38,10 @@ message RuntimeInfoReport {
   bytes git_hash = 7; //the git hash of the build related the git repository's branch
   bytes version = 8;
   bytes db_status_extra = 9; //RESERVED: JSON format structure to store db status information
-  bytes queue_status_extra = 10; //RESERVED: JSON format structure to store the queue information
-  bytes chain_status_extra = 11; //RESERVED: JSON format structre to store the block chain related information
-  bytes cache_status_extra = 12; //RESERVED: JSON format structre to store the cache related information
-  bytes extra = 13; //RESERVED: JSON format structre to store other information
-  bytes signature = 14;
+  bytes rd_status_extra = 10; //RESERVED: JSON format structure to store redis status information
+  bytes queue_status_extra = 11; //RESERVED: JSON format structure to store the queue information
+  bytes chain_status_extra = 12; //RESERVED: JSON format structre to store the block chain related information
+  bytes cache_status_extra = 13; //RESERVED: JSON format structre to store the cache related information
+  bytes extra = 14; //RESERVED: JSON format structre to store other information
+  bytes signature = 15;
```

Comment:
The "bytes signature = 15;" im RuntimeInfoReport may also need to move out like the SignedRuntimeInfoRequest.
How do you think @wxue @JustinGaoF

Reply:
@wxue @JustinGaoF any ideas ?

Reply:
Good catch. I don't think we need to sign RuntimeInfoReport since it will be used internally.

Reply:
ok